frontend/javascripts/oxalis/model/sagas/mesh_saga.ts

```diff
@@ -1087,21 +1106,25 @@ function* downloadMeshCellsAsZIP(
   const additionalCoordinates = yield* select((state) => state.flycam.additionalCoordinates);
   try {
     const addFileToZipWriterPromises = segments.map((element) => {
-      const geometry = segmentMeshController.getMeshGeometryInBestLOD(
-        element.segmentId,
-        element.layerName,
-        additionalCoordinates,
-      );
+      if (
+        segmentMeshController.hasMesh(element.segmentId, element.layerName, additionalCoordinates)
+      ) {
```

Comment:
why is this check needed? couldn't getMeshGeometryInBestLOD return null if hasMesh is false? currently, the user wouldn't see any toast, if this outer if evaluates to false which is not optimal.

Reply:
Before this, nothing would be downloaded, because `getMeshGeometryInBestLOD` used to throw an error and nothing was downloaded. So if one of the segments doesn't have a mesh, there should be a toast, and the available should be downloaded, right?

Reply:
I am realizing that I was mixing up bugs. The malfunctioning mesh download still stems from the CSP. I'll add a null check in `getMeshGeometryInBestLOD` and revert the other changes